### PR TITLE
DEV-4317: days_and_numbers

### PR DESCRIPTION
**Pull request by MSilva95**

### CODE REVIEW

File: subjects/devops/days_and_numbers/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+## Days and numbers
+
+### Instructions
+
+Create a file `custom_calendar.py` which will have 2 functions:
+
+- `day_from_number(day_number)`
+- `day_to_number(day)`
+
+Those functions perform conversion between day `index` and day `word` and vice versa:
```
Comment (mikysett) [inline subjects/devops/days_and_numbers/README.md:10]: "and vice versa" is redundant since "conversion between" is already a two way road.

**Review by mikysett — CHANGES_REQUESTED** [review]
- little style improvements
- small error about the fact that dictionary are ordered or not

PS: I've noticed you usually leave the default description into the PR, please remove it if you don't have a description, otherwise we have a big default text at the start of the PR which doesn't actually describe the issue

File: subjects/devops/days_and_numbers/README.md (hunk 1 above)
Comment (MSilva95) [reply subjects/devops/days_and_numbers/README.md:10]: done

File: subjects/devops/days_and_numbers/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+```
+
+```console
+$ python test.py
+Tuesday
+Monday
+None
+7
+None
+```
```
[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/devops/days_and_numbers/README.md:44]: done

File: subjects/devops/days_and_numbers/README.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+```python
+phonebook = {
+    "John" : 938477566,
+    "Jack" : 938377264,
+    "Jill" : 947662781
+}
+print(phonebook)
+```
+
+Dictionaries can be iterated over, just like a list. However, a dictionary, unlike a list, does not keep the order of the values stored in it. To iterate over key value pairs, use the following syntax:
```
[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/devops/days_and_numbers/README.md:79]: done

File: subjects/devops/days_and_numbers/README.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+You should return `None` if the input is invalid (invalid number or day string).
+
+### Usage
+
+Here is a possible `test.py` to test your functions:
+
+```python
+import custom_calendar
+
+if __name__ == '__main__':
```
[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/devops/days_and_numbers/README.md:29]: Done! thanks for the tip!

**Review by mikysett — APPROVED** [review]
LGTM

**Pull request merged**